api/swagger.yaml

```diff
@@ -0,0 +1,788 @@
+    DecommissionCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+    RenameCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        - name
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          name:
+            type: string
+    AttackCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - id
+        - target
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          target:
+            type: integer
+            format: int64
+    TradeCommand:
+      allOf:
+      - $ref: '#/components/schemas/Command'
+      - required:
+        - amount
+        - id
+        - resource
+        - where
+        type: object
+        properties:
+          id:
+            type: integer
+            format: int64
+          resource:
+            type: integer
+            format: int64
+          amount:
+            type: integer
+            format: int64
+          where:
+            type: integer
+            format: int64
+    Commands:
+      type: object
+      properties:
+        commands:
+          type: array
+          items:
+            $ref: '#/components/schemas/Command'
+    Credentials:
+      required:
+      - name
+      - password
+      - player
+      type: object
+      properties:
+        name:
+          type: string
+        password:
+          type: string
+        player:
+          type: string
+    Tick:
+      type: object
+      properties:
+        tick:
+          type: integer
+          description: Game tick in milliseconds
+          format: int64
+          example: 10
+        wait:
+          type: boolean
+          default: true
+
+    Coordinates:
+      type: object
+      properties:
+        x:
+          type: integer
+          format: int64
+        y:
+          type: integer
+          format: int64
+      example:
+        x: 10
+        y: 20
+
+    StaticData:
+      type: object
+      properties:
+        classes:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              name:
+                type: string
+              shipyard:
+                type: boolean
+              speed:
+                type: integer
+                format: int64
+              cargo:
+                type: integer
+                format: int64
+              life:
+                type: integer
+                format: int64
+              damage:
+                type: integer
+                format: int64
+        resources:
+          type: object
+          additionalProperties:
+            type: string
+      example:
+        classes: {
```

Comment:
maybe we can rename this to `ship_classes` and in analogy `resource_names`

Reply:
Done